render_block/django.py

```diff
@@ -10,9 +10,13 @@
 from render_block.exceptions import BlockNotFound
 
 
-def django_render_block(template, block_name, context):
+def django_render_block(template, block_name, context, request=None):
     # Create a Django Context.
-    context_instance = Context(context)
+    if request:
+        context_instance = RequestContext(request)
```

Comment:
I think this can take the rest of the context as a second input parameter, so this would be clearer as:
`context_instance = RequestContext(request, context)`. (I'll make this change if I don't hear back!)

Reply:
Nice catch, fixed!